Repositories/ItemRepository/ItemRepository.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DomainModel;
+using System.Linq;
+
+namespace Repositories.ItemRepository
+{
+    public class ItemRepository : IItemRepository
+    {
+        DataContext context;
+
+        public ItemRepository(DataContext datacontext)
+        {
+            context = datacontext;
+        }
+
+        public IQueryable<Item> All()
+        { 
+            return context.Items.AsQueryable();
+        }
+
+        public void Delete(int id)
+        {
+            context.Items.RemoveAt(id - 1);
```

Comment:
I think this is not an ideal way of removing items. The id is not comparable to an index.